WordPress/WordPressTest/ActivityLogRangesTest.swift

```diff
@@ -0,0 +1,108 @@
+import XCTest
+@testable import WordPress
+
+final class ActivityLogRangesTests: XCTestCase {
+
+    let testData = ActivityLogTestData()
+
+    override func setUp() {
```

Comment:
Would it make sense to remove these overrides?

Reply:
Absolutely!